x-pack/plugins/ml/public/maps/util.ts

```diff
@@ -23,6 +23,33 @@ export const ML_ANOMALY_LAYERS = {
 } as const;
 
 export type MlAnomalyLayersType = typeof ML_ANOMALY_LAYERS[keyof typeof ML_ANOMALY_LAYERS];
+const INFLUENCER_LIMIT = 3;
+const INFLUENCER_MAX_VALUES = 3;
+
+function getInfluencersHtmlString(
```

Comment:
nit: Would be good to add some unit tests for this utility function

Reply:
Added in 1dd59765c032b18dd9de60c53f9a1c24e86cf5d4